Moodboard-program/moodboard-prototype

```diff
@@ -0,0 +1,262 @@
+import sys
+import time
+
+
+
+def paintblack(canvas2):
+  pixels = getPixels(canvas2)
+  for i in pixels:
+    setColor(i,black)
+    
+
+def start():
+  #load image from file
+  canvas2=makePicture(r".\Ship-image-structure\ship+biodomes.jpg")
+  show(canvas2)
+  #get input from user
+  
+  keypress = requestNumber("Press 1 for biodome or 2 for ship: ")
+  #if the user entered 1 then zoom into the top section of the image
+  #if the user enters 2(or anything else) zoom into the bottom part of the image
+  
+  if keypress == 1:
+    paintblack(canvas2)
+    repaint(canvas2)
+    EnlargeImageTopSection()
+    time.sleep(2)
+  elif keypress == 2:
+    paintblack(canvas2)
+    repaint(canvas2)
+    EnlargeImageBottomSection()
+    time.sleep(2)
+
+
+def EnlargeImageTopSection():
+  #get the image and then make an empty image 
+  file=(r".\Ship-image-structure\ship+biodomes.jpg")
+  src=makePicture(file)
+  
+  canvas = makeEmptyPicture(1326,698)
+  canvas2 = makeEmptyPicture(1326,698)
+  
```

Comment:
Perhaps use more descriptive variable names, as it is hard to know what 'canvas' and 'canvas2' are without reading through the code